lib/domain/libBlockchain/langs/yul/YulModel.h

```diff
@@ -34,24 +34,17 @@ namespace vanguard {
 
             return regex_match(llvmFn.getName().str(), reg) && blockchainFn.params().size() + 1 == llvmFn.arg_size();
         }
-        //bool isExternalCall(const llvm::Function &llvmFn) override;
-        //bool isDelegateCall(const llvm::Function &llvmFn) override;
-        bool isAnyLowLevelCall(CallIns<Domain> &call) const override {
-            return isLowLevelCall(call) || isLowLevelStaticCall(call) || isLowLevelDelegateCall(call);
-        }
 
-        bool isLowLevelCall(CallIns<Domain> &call) const override {
-            //throw std::runtime_error("To be implemented");
-            return false;
+        bool isCall(const CallIns<Domain> &call) const override {
+            // https://github.com/Veridise/yul2llvm/blob/a545bc791de052d0984898fd594846d8e144cdb4/lib/libyul2llvm/YulASTVisitor/IntrinsicRewriter.cpp#L71-L126
+            return callNameMatches(call, std::regex("pyul_call(_unknown|0x[0-9a-fA-F]{8})?"));
```

Comment:
nit: this will compile the regex every time `isCall()` is invoked. We can make it `const static` so that the regex is only compiled once:
```suggestion
            const static std::regex callRegex("pyul_call(_unknown|0x[0-9a-fA-F]{8})?");
            return callNameMatches(call, callRegex);
```